js/commands.js

```diff
@@ -59,6 +59,20 @@ $(function(){
     }
   });
 
+  // Need to use keyup to correctly capture length of .cmd-field
+  $('.cmd-field').keyup(function(e){
+    // Backspace key and Delete key
+    if (e.which == 8 || e.which == 46){
+      var command = $(this).val();
+
+      if (command.length == 0 && !main.isStarted){
```

Comment:
Oh. I hadn't realised you could still access `.cmd-input` in-game. Wouldn't it be better to disable text-input when in-game, instead? I'll still be removing the condition, nonetheless.